tools/fastdds/fastdds.py

```diff
@@ -42,7 +42,10 @@
 """
 
 import argparse
+import pathlib
 import sys
+import textwrap
+import xml.etree.ElementTree as ET
```

Comment:
```suggestion
import sys
```
We are not using these libraries anymore.